test/integration/content-explorer/ContentExplorer.e2e.test.js

```diff
@@ -0,0 +1,207 @@
+// <reference types="Cypress" />
+const helpers = {
+    load(additionalProps = { canUpload: true, permissions: { canUpload: true } }) {
+        cy.visit('/Elements/ContentExplorer', {
+            onBeforeLoad: contentWindow => {
+                contentWindow.PROPS = additionalProps;
+            },
+        });
+    },
+    getRow(rowNum) {
+        return cy.getByDataPreview('ContentExplorer').find(`.bce-item-row-${rowNum}`);
+    },
+    checkRows(rowNum) {
```

Comment:
Maybe rename the method to be more explicit about what it does, perhaps `checkRowSelections` and `rowNum` => `selectedRow`